migrations/20191125132858_get-messaging-service-type.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function(knex) {
+  /*
+   * Note – this function should be actually replaced in production
+   * with values depending on the particular migration context
+   */
+  return knex.schema.raw(`
+    create or replace function get_messaging_service_type(zip text) returns messaging_service_type as $$
```

Comment:
The way that subsequent queries are written, since this happens on the cell level, not the campaign contact level, this is a non trivial increase in complexity for flexibility that we don't need yet (and might not)